### ISSUE TITLE
Daemon Crashed

### ISSUE THREAD

**Issue by ilhamm24** [MinaProtocol/mina#6626]
Hi, i have problem with message 
```[2020-11-9 15:32:43.823504]Init__Coda_run: Successfully wrote the visualization of the registered masks at location: /root/.coda-config/coda_crash_report_2020-11-09_14-32-43.823431/registered_masks.dot.
[2020-11-9 15:32:43.828335]Block_time: Environment variable CODA_TIME_OFFSET not found, using default of 0
!!!
!!!
!!!   ☠  Coda Daemon crashed.
!!!    The Coda Protocol developers would like to know why!
!!!
!!!     Please:
!!!       Open an issue:
!!!         <https://github.com/CodaProtocol/coda/issues/new>
!!!
!!!       Briefly describe what you were doing and attach the crash report /root/.coda-config/coda_crash_report_2020-11-09_14-32-43.823431.tar.gz
!!!
!!!
```

**Comment by garethtdavies** [comment]
@ilhamm24 You need to provide the crash report, or the logs preceding the crash, as there isn't sufficient information to go on from your post.

**Comment by ilhamm24** [comment]
Sorry, here's coda.log
[coda.log](https://github.com/MinaProtocol/mina/files/5511755/coda.log)

**Comment by ilhamm24** [comment]
Also status.log
[status.log](https://github.com/MinaProtocol/mina/files/5511765/status.log)

**Comment by aneesharaines** [comment]
Fixed by #6629
